packages/appChrome/stories/AppChrome.stories.tsx

```diff
@@ -19,7 +19,7 @@ export default {
   component: AppChrome
 } as Meta;
```

Comment:
Do we want Storybook users to be able to use the controls for the AppChrome props? If not, should we disable them? At the minute they're not very intuitive, and also, clicking on the `Set object` button under `App Chrome Bare` gives an error.

Reply:
Yeah, agreed that it's not very helpful when you have to pass in an object of some sort. It's also difficult to have controls for nested aspects which is why I haven't included the color picker for the section below. I'm going to disable controls for this story.